From 2ba7b9ea395bcf28a72bc684a5884375b555bbc1 Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <dinguyen@altera.com>
Date: Wed, 19 Mar 2014 14:48:11 -0500
Subject: [PATCH 200/248] FogBugz #192127: Fix CTI address length

The register base length for the CTI module should be 0x1000, not 0x100.

[Original patch taken from
git://git.rocketboards.org/linux-socfpga.git socfpga-3.14]

Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
Signed-off-by: Dinh Nguyen <dinguyen@altera.com>
---
 arch/arm/boot/dts/socfpga.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/socfpga.dtsi b/arch/arm/boot/dts/socfpga.dtsi
index 10fa771..29ef6f5 100644
--- a/arch/arm/boot/dts/socfpga.dtsi
+++ b/arch/arm/boot/dts/socfpga.dtsi
@@ -697,12 +697,12 @@
 
 			cti0: cti0@ff118000 {
 				compatible = "arm,coresight-cti";
-				reg = <0xff118000 0x100>;
+				reg = <0xff118000 0x1000>;
 			};
 
 			cti1: cti1@ff119000 {
 				compatible = "arm,coresight-cti";
-				reg = <0xff119000 0x100>;
+				reg = <0xff119000 0x1000>;
 			};
 		};
 
-- 
1.9.1

